src/carts/src/main/java/mushop/carts/CartRepositoryDatabaseImpl.java

```diff
@@ -141,5 +146,18 @@ public void save(Cart cart) {
             throw new RuntimeException(e);
         }
     }
+
+    @Override
+    public boolean healthCheck(){
+        try (Connection con = pool.getConnection()) {
+            OracleDatabase db = SODA.getDatabase(con);
+            OracleCollection col = db.openCollection(collectionName);
+            String name = col.admin().getName();
+            return name == null ? false : true;
```

Comment:
Health check just return true or false? no payload? This case does not return health check for the application?

Reply:
Helidon does support adding a payload, but the method that returns the boolean is simply used to construct a standard payload (with just status messges.) The Builder for the healthcheck will use the method to determine the up/down status and then a separate paylaod method to create payload. There wasn't really any thing here that I could use for a a better payload. It does however use JVM metrics for the health as well in addition to the "custom" DB healthcheck. the response from /health will look like this: 
The states and logical `AND`ed to arive at the overall health, which also determines the HTTP response  code (200=OK, 503=unhealthy, 500 = healthcheck itself had some error, so app status is unknown) 
```
{
    "outcome": "UP",
    "status": "UP",
    "checks": [
        {
            "name": "dbHealth",
            "state": "UP",
            "status": "UP"
        },
        {
            "name": "deadlock",
            "state": "UP",
            "status": "UP"
        },
        {
            "name": "diskSpace",
            "state": "UP",
            "status": "UP",
            "data": {
                "free": "19.35 GB",
                "freeBytes": 20781621248,
                "percentFree": "33.13%",
                "total": "58.41 GB",
                "totalBytes": 62722478080
            }
        },
        {
            "name": "heapMemory",
            "state": "UP",
            "status": "UP",
            "data": {
                "free": "23.14 MB",
                "freeBytes": 24260408,
                "max": "128.00 MB",
                "maxBytes": 134217728,
                "percentFree": "68.08%",
                "total": "64.00 MB",
                "totalBytes": 67108864
            }
        }
    ]
}
```